lib/rofl_ofdpa_fm_driver.cpp

```diff
@@ -295,7 +295,8 @@ cofflowmod rofl_ofdpa_fm_driver::enable_tmac_ipv4_unicast_mac(
   fm.set_cookie(
       gen_flow_mod_type_cookie(OFDPA_FTT_TERMINATION_MAC_IPV4_UNICAST_MAC) | 0);
 
-  fm.set_match().set_in_port(in_port);
+  if (in_port)
+    fm.set_match().set_in_port(in_port);
```

Comment:
from API point of view, I would prefer making this more visible, like adding default parameter port=0

Reply:
This is the same behaviour that we have in other functions like enable_policy_acl_ipv4_vlan. Since there is no good default parameter I am not sure how to make this more visible.

Reply:
Having said that I know we could add a separate call to this, but this would lead into an explosion of functions.